kernel/libs/aster-bigtcp/src/iface/poll.rs

```diff
@@ -456,7 +472,9 @@ impl<E: Ext> PollContext<'_, E> {
             let mut deferred = None;
 
             socket.dispatch(self.iface_cx, |cx, ip_repr, udp_repr, udp_payload| {
-                let mut this = PollContext::new(cx, self.sockets, self.new_tcp_conns);
+                let mut dead_conns = Vec::new();
+                let mut this =
+                    PollContext::new(cx, self.sockets, self.new_tcp_conns, &mut dead_conns);
```

Comment:
Add a `debug_assert!(dead_conns.is_empty())` as well as a comment explaining that we are dealing with UDP sockets, so there can't be any dead TCP sockets?

(This seems _quite_ obvious, but it's still beneficial for code clarity).